### PR TITLE
Update overview-secure-score.md

### PR DESCRIPTION
**Pull request by ojrb**
This PR solved the issue #3738, I add the times for the Update Schedule for Secure Score, @e0i please review and merge !

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-atp/overview-secure-score.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ In the example image, the total points for the Windows security controls and Off
 You can set the baselines for calculating the score of Windows Defender security controls on the Secure score dashboard through the **Settings**. For more information, see [Enable Secure score security controls](enable-secure-score-windows-defender-advanced-threat-protection.md).
 
 ## Secure score over time
-You can track the progression of your organizational security posture over time using this tile. It displays the overall score in a historical trend line enabling you to see how taking the recommended actions increase your overall security posture.
+You can track the progression of your organizational security posture over time using this tile. It displays the overall score in a historical trend line enabling you to see how taking the recommended actions increase your overall security posture.  The expected updated schedule for the Secure Score is about 24 hours, but in some cases depending of the size of the organization, numbers of computers and other factors, this update can be at least 72 hours.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/overview-secure-score.md]:
```suggestion
You can track the progression of your organizational security posture over time using this tile. It displays the overall score in a historical trend line enabling you to see how taking the recommended actions increase your overall security posture.  The expected update schedule for Secure Score is about 24 hours. In some cases, depending of the size of the organization, number of computers and other factors, this update can take up to 72 hours.
```

**Comment by e0i** [comment]
@ojrb Please follow-up with suggested copy editing improvements. Thanks.

**Comment by ojrb** [comment]
Ready I accepted the suggestions !

**Comment by e0i** [comment]
@mjcaparas The PR that addresses issue #3738 and has passed copy review is ready for technical review. Thanks.

**Pull request merged**